snippets/vhdl.yml

```
@@ -136,6 +136,60 @@ for_generate:
     - "\t$0"
     - "end generate $1;"
 
+# --------------------------------------------------
+# Process
+# --------------------------------------------------
+
+testbench_process:
+  prefix: tproc
+  description: Testbench Process (No Sensitivity List) 
+  scope: source.vhdl
+  body:
+    - "${1:proc_name}: process"
+    - "begin"
+    - "$0"
+    - "end process; -- $1"
```

The label can be used before the semi colon, which makes more sense than an automatic comment:
```suggestion
    - "end process $1;"
```

---

You're right, that must have been a typo on my end.

---

This one still shows up unchanged at my end, all the others are fine though.

---

Forgot it, committing now.